views/viewEvent.ejs

```diff
@@ -31,24 +33,26 @@
           <p class="card-text"><%= locals.event.description %></p>
           <a href="#" class="btn btn-primary">url: <%= locals.event.url %></a>
         </div>
-        <div class="card-footer text-muted">
+        <!-- No footer info atm -->
+        <!-- <div class="card-footer text-muted">
           footer some agregate data - total attendees / days till event
-        </div>
+        </div> -->
       </div>
     </section>
     <section>
       <table id="eventTimesTable" class="table table-bordered">
         <thead>
           <tr>
-            <th scope="col">#</th>
+            <th scope="col">Dates and Times</th>
           </tr>
         </thead>
         <tbody>
           <tr>
           </tr>
         </tbody>
       </table>
-      <button id="eventAddParticipant">+ Add Participant</button>
+      <!-- No longer need to add as user is created before -->
+      <!-- <button id="eventAddParticipant">Add Participant</button> -->
     </section>
   </main>
```

Comment:
the branch I pushed at 9:00PM last night substantially changes this file to implement server side rendering of this page.